docs/conf/eu/2017/visiting.rst

```diff
@@ -0,0 +1,140 @@
+:template: 2017/eu-content.html
+
+Visiting Prague
+===============
+
+Wondering where to stay and how to get around once you get into Prague?
+
+Traveling to Prague
+-------------------
+
+Prague has many direct rail, bus, and flight connections to most major
+European cities. In the summer, there are also direct transatlantic
+flights to/from New York (JFK) and several other US airports.
+
+Flights
+~~~~~~~
+
+Prague's `Vaclav Havel airport <http://www.prg.aero/en/>`__ provides
+many direct and low-cost flights with airlines such as Czech Airlines,
+Ryanair, Wizz Air, Air Berlin, Norwegian... You get the idea. To get
+from the airport to the city center, you can either take a bus (average
+joureny time 30 minutes, cost ~30czk) or a taxi (average journey time
+15-20 minutes depending on traffic, cost ~500czk).
+
+Trains
+~~~~~~
+
+Prague's main train station (Hlavni Nadrazi / hl.n) is a central railway
+hub, with direct trains to Germany, Poland, Austria, and Hungary. We
+recommend train lines by EuroCity (EC), InterCity (IC) or RailJet (RJ).
+You can even take a train from Denmark `inside a
+ferry! <http://en.wikipedia.org/wiki/Vogelfluglinie>`__
+
+To look up train connections, head over to the `ČD
+Website <https://www.cd.cz/eshop/default.aspx>`__.
+
+Buses
+~~~~~
+
+Prague is a popular destination among backpackers and shoestring
+travelers, so there are many affordable bus options if you are traveling
+within Europe. The biggest Czech bus company `Student
+Agency <http://www.studentagency.eu/>`__ operates direct bus lines
+between Prague and cities like Berlin, Vienna, Zagreb, Munich, etc.
+Global companies like `Eurolines <http://www.eurolines.com/en/>`__
+operate direct lines to many European cities.
+
+How to get around
+-----------------
+
+Prague is a very accessible city and there are many transportation
+options available, public and otherwise.
+
+Navigating in Prague
+~~~~~~~~~~~~~~~~~~~~
+
+If you navigate by smartphone, Google Maps is the most popular method to
+calculate routes and travel times. Do exercise caution when calculating
+walking routes, as the city center has many gated walkways that are
+locked at night.
+
+Trams, buses, and metro
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The `Prague Public Transit Co. <http://www.dpp.cz/en/>`__ runs an
+extensive network of trams, buses, and 3 metro lines. You can buy a
+single ticket, day ticket, or 5-day pass at ticket machines, Trafika
+stands (tobacconists) and at the train and bus stations.
+
+If you're flying into Prague Airport (PRG), you can take bus number 100,
+119, or the Airport Express (AE) bus. Depending on where you are
+staying, these bus lines will connect you to several major metro
+stations as well as the main train station. `More information on airport
+transfers. <http://www.dpp.cz/en/public-transit-to-prague-airport/>`__
+
+Taxi companies
+~~~~~~~~~~~~~~
+
+Prague is serviced by **Uber**, and it's the most reliable and cheapest way to secure
```

Comment:
Yeah, I'm talking from a political perspective not from an end user service perspective. I suspect it will be most people's goto anyway, so we don't need to actually endorse them.